SmartDeviceLink/SDLTouchManager.m

```diff
@@ -212,6 +214,19 @@ - (void)sdl_onTouchEvent:(SDLRPCNotificationNotification *)notification {
     }];
 }
 
+- (SDLOnTouchEvent *)applyScaleToEventCoordinates:(SDLOnTouchEvent *)onTouchEvent {
+    float scale = self.videoStreamingCapability.scale.floatValue;
+    if (scale > 0) {
+        for (SDLTouchEvent *touchEvent in onTouchEvent.event) {
+            for (SDLTouchCoord *coord in touchEvent.coord) {
+                coord.x = @(coord.x.floatValue / scale);
+                coord.y = @(coord.y.floatValue / scale);
+            }
+        }
+    }
+    return onTouchEvent;
+}
+
```

Comment:
In general I think that copying it before modification and returning the modified copy is probably a better way to go. Since the RPC can be returned elsewhere if the developer adds their own RPC listeners, it's better to copy it if you're going to modify it.

Reply:
Hello @kshala-ford, @joeljfischer. Done.